recipes/qpid-proton/all/conanfile.py

```diff
@@ -0,0 +1,144 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.microsoft import is_msvc
+
+import os
+
+required_conan_version = ">=2.1.0"
+
+class QpidProtonConan(ConanFile):
+    version = "0.40.0"
```

Comment:
```suggestion
```
Avoid using explicit version in the recipe. It should be configured by `conan create` command and populated by conandata.yml data.